src/ProjectsClient.ts

```diff
@@ -39,21 +39,21 @@ export class ProjectsAccessClient implements ProjectsAccess {
    * @returns Array of projects meeting the query's requirements
    */
   private async getByQuery(accessToken: AccessToken, queryArg?: ProjectsQueryArg): Promise<Project[]> {
-    const requestOptions: RequestOptions = this.getRequestOptions(accessToken);
+    const requestOptions = this.getRequestOptions(accessToken);
     let url = this._baseUrl;
     if (queryArg)
       url = url + this.getQueryString(queryArg);
 
     const projects: Project[] = [];
 
     try {
-      const response = await request(url, requestOptions);
+      const response = await axios.get(url, requestOptions);
```

Comment:
couldn't we just use fetch?

Reply:
wanted to make sure it would work in node/backend code as well.

Reply:
ah gotcha